content/en/docs/setup/quickstart.md

```diff
@@ -132,6 +132,30 @@ Content-Length: 0
 
 Note: 10.244.0.21 is the IP of httpbin
 
+## Enable dataplane bpf log
```

Comment:
@bfforever Would you update the design as we talked, if so this need update

Reply:
I will update next week.